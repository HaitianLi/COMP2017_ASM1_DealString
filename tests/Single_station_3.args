Sydney Melbourne 08:00:00
