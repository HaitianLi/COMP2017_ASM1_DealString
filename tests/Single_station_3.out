The next train to Melbourne from Sydney departs at 08:05:00
